### DISCUSSION TITLE
Practice only on error word

### DISCUSSION THREAD

**Discussion by wztdream** [monkeytypegame/monkeytype#2105] [Ideas]
Hi, 
Maybe this function is already there. I think it will be helpful to have a mode that only show the words the user have made mistake before. This is a kind of learning from mistake. I notice for myself, I made mistake mostly on several letter, like "public","think", so if we allow the user to  focus the attention only on these high mistake words, it may improve the typing quality more efficiently.

Another idea is to allow user to manually add some  words to a "mistaken word list", and allow users to practice on them based on the error frequency.

**Comment by rpnfan** [comment]
Would still be helpful indeed. Other websites like typerfast.com and typecelerate.com are now taking this to the next level.

**Comment by Miodec** [comment]
On the result screen you can press the "Practice missed words" button.
